### PR TITLE
test: increase coverage (OZ M-02)

### PR DESCRIPTION
**Pull request by tmigone**
```
| File                        | % Lines          | % Statements     | % Branches     | % Funcs        |
|-----------------------------|------------------|------------------|----------------|----------------|
| src/AllocationIDTracker.sol | 100.00% (10/10)  | 100.00% (13/13)  | 100.00% (4/4)  | 100.00% (3/3)  |
| src/Escrow.sol              | 98.15% (106/108) | 98.35% (119/121) | 97.37% (37/38) | 94.12% (16/17) |
| src/TAPVerifier.sol         | 80.00% (4/5)     | 66.67% (6/9)     | 100.00% (0/0)  | 66.67% (2/3)   |
| Total                       | 97.56% (120/123) | 96.50% (138/143) | 97.62% (41/42) | 91.30% (21/23) |
```

### CODE REVIEW

**Review by aasseman — COMMENTED** [review]
LGTM. Will let @ColePBryan do the actual approval.

File: test/Escrow.t.sol [hunk 1]
```diff
@@ -491,6 +491,30 @@ contract EscrowContractTest is Test {
...
+        // Sets msg.sender address for next contract calls until stop is called
+        vm.startPrank(SENDER_ADDRESS);
+        vm.expectRevert(
+            abi.encodeWithSignature(
+                "InsufficientEscrow(uint256,uint256)", 
+                ESCROW_AMOUNT, 
+                100*ESCROW_AMOUNT
+            )
+        );
+        escrowContract.thaw(receiversAddresses[0], 100*ESCROW_AMOUNT);
```
[the remark this answers is not in the document]
Comment (tmigone) [reply test/Escrow.t.sol]: good idea, thats a better value for the test

File: test/Escrow.t.sol [hunk 2]
```diff
@@ -53,6 +53,59 @@ contract EscrowContractTest is Test {
...
+        address indexed authorizedSigner
+    );
+    event Withdraw(
+        address indexed sender,
+        address indexed receiver,
+        uint256 amount
+    );
+    event AuthorizeSigner(address indexed signer, address indexed sender);
+    event Transfer(address indexed from, address indexed to, uint256 value);
+
```
Comment (ColePBryan) [inline test/Escrow.t.sol:108]: Is it possible to import the events instead of redefining them? It seems like this could make maintenance difficult.

Comment (tmigone) [reply test/Escrow.t.sol:108]: Unfortunately there isn't a clean way of doing it. Best thing I can think of is to define an interface for all the events then inherit it on both the contract and the test contract, but i'm not too happy with that solution. 

Foundry seems to recommend just re declaring the events: https://book.getfoundry.sh/cheatcodes/expect-emit, which I agree is also not good due to maintenance and duplicate code.

**Review by ColePBryan — APPROVED** [review]

**Pull request merged**